### DISCUSSION TITLE
Documentation Revamp

### DISCUSSION THREAD

**Discussion by oschuett** [cp2k/cp2k#2819] [Show and tell]
Hi Everybody,

as discussed at the last developers meeting, I'd like to improve our documentation.
As a first step I've create a new version of our input reference: https://manual.cp2k.org/trunk/new/
Please take a look. If there are no problems, I would like to switch over after the next release.

**Comment by mkrack** [comment]
Great work @oschuett, modern layout and properly searchable now.

**Reply by oschuett** [reply]
Thanks! Yes, I was also pleasantly surprised by Sphinx's builtin search. My original plan was to use [Algolia](https://docsearch.algolia.com), but that does no longer seem worth the effort (nor adding an external dependency) .
